### ISSUE TITLE
Not an issue, just a request after the new version

### ISSUE THREAD

**Issue by AuntyT** [Drahsid/HybridCamera#9]
Hello, it seems that the customizable WASDEQ buttons have merged into 1 option which allows the A and D rotate the camera. Would it be possible to have the option to customize the buttons as they were before? Thank you for reading

**Comment by Symphunis** [comment]
Yes, I love standard type, but just want my character to run backwards instead of walk backwards. Being able to keybind it to specific keys was wonderful. Please add this back. :)

**Comment by Symphunis** [comment]
> > Yes, I love standard type, but just want my character to run backwards instead of walk backwards. Being able to keybind it to specific keys was wonderful. Please add this back. :)
> 
> I actually made standard-style backpedaling full speed before 7.1 (see commit [c211e25a531bf7a9b9d47dd6a4ea5cdae1494315](https://github.com/Drahsid/HybridCamera/commit/c211e25a531bf7a9b9d47dd6a4ea5cdae1494315)), however, I was asked to remove the feature to make the plugin legal. That said, you can technically get full-speed behavior by toggling "Use turn on backpedal", the difference being you don't face the camera when moving this way.

The only issue with that is that it only works while in Legacy. Much like Narcasung, I'm looking for WAD standard and S legacy. It's less about the speed and more for being able to quickly turn around with a button rather than swinging my camera. Regardless, I still appreciate all you do!

**Comment by Narcasung** [comment]
> I was asked to remove the feature to make the plugin legal

That explains it, it's true that having standard WAD and legacy S is very powerful imo. There's no way around it if you wanna stay on the official repo I guess. I'll just get used to strafing again.
